server/routes/api/articles.js

```diff
@@ -19,7 +22,7 @@ router.post('/:article_slug/comments/:commentId/:reaction', authenticateUser, Co
 router.get('/:article_slug/comments/:commentId/reaction_status', authenticateUser, CommentsController.checkReactionStatus);
 router.get('/:article_slug/comments', authenticateUser, CommentsController.getArticleComments);
 router.put('/:article_slug/comments/:commentId', authenticateUser, CommentsController.updateComment);
-router.get('/', authenticateUser, ArticleController.getAllArticles);
+router.get('/', ArticleController.getAllArticles);
```

Comment:
@emekafredy Why was `authenticateUser` removed?

Reply:
We need to populate the landing page with articles for users to see, whether logged in or not. This function doesn't return the article body, but only the title, description and slug. I f they decided to now click on an article to read, they call the getSingleArticle function, which will then require authentication

Reply:
👍